src/components/generic/movieModal/modalContent.tsx

```diff
@@ -109,26 +112,26 @@ export default ({
                     height="250"
                     src={`https://www.youtube.com/embed/${trailer}`}
                     frameBorder="0"
-                    mozallowfullscreen="mozallowfullscreen"
-                    msallowfullscreen="msallowfullscreen"
-                    allowFullScreen="allowFullScreen"
-                    webkitallowfullscreen="webkitallowfullscreen"
+                    allowFullScreen
+                    
                     allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
                   />
                 }
               />
               <SelectQuality links={items} imdb={imdb} />
             </Modal.right>
-          </Modal.content>
-        </Modal>,
+          </Modal.ModalContent>
+        </Modal.Modal>,
       ]}
     </PoseGroup>
   )
 }
 
-const Category = ({title, render}) => (
-  <Modal.center.category>
-    <Modal.center.category.title>{title}</Modal.center.category.title>
+export default ModalContent
+
+const Category: FC<{ title: string, render: React.ReactChild }> = ({ title, render }) => (
```

Comment:
Why not use `children`? Creating an additional prop seems unnecessary.

Reply:
:white_check_mark: